pkg/env/dev.go

```diff
@@ -297,3 +300,18 @@ func (d *dev) CreateARMResourceGroupRoleAssignment(ctx context.Context, fpAuthor
 		return true, nil
 	})
 }
+
+func (d *dev) E2EStorageAccountName() string {
+	return "arov4e2eint"
+}
+
+func (d *dev) E2EStorageAccountKey(ctx context.Context) (string, error) {
+	if d.e2eStorageAccountKey == "" {
+		keys, err := d.e2estorage.ListKeys(ctx, "global-infra", d.E2EStorageAccountName(), "")
+		if err != nil {
+			return "", err
+		}
+		d.e2eStorageAccountKey = *(*keys.Keys)[0].Value
+	}
+	return d.e2eStorageAccountKey, nil
+}
```

Comment:
I feel this would be stronger if the ListKeys moved to a constructor function in pkg/util/billing.  This would mean that if it fails, we error out at startup, which is a clearer indicator of trouble.

I am just about OK with defining constants like "global-infra", etc., in env functions, but I feel they would be much better passed in as RP configuration.